python/src/opendp/prelude.py

```diff
@@ -49,4 +50,4 @@
 from opendp.context import *
 from opendp.polars import *
 
-__all__ = ["t", "m", "c"]
+__all__ = ["sklearn", "x", "t", "m", "c"]
```

Comment:
I've been a bit uncomfortable with our existing short namespaces, and would prefer to spell things out. `np` and `pl` are the conventional import names for their respective libraries, but they are just conventions. 

Otherwise, having them accessible from the prelude seems fine to me:

```
dp.numpy.array2_domain
dp.sklearn.PCA
dp.polars.DPExpr
```

Thoughts?

Reply:
This makes sense. Let's do this for extras.